app/main.py

```diff
@@ -1,3 +1,63 @@
+from __future__ import division
+
+
 class Distance:
-    # Write your code here
-    pass
+
+    def __init__(self, km):
+        self.km = km
+
+    def __str__(self):
+        return f"Distance: {self.km} kilometers."
+
+    def __repr__(self):
+        return f"Distance(km={self.km})"
+
+    def __add__(self, other):
+        if isinstance(other, int):
+            return Distance(self.km + other)
+        return Distance(self.km + other.km)
+
+    def __iadd__(self, other):
+        if isinstance(other, int):
```

Comment:
```suggestion
        if isinstance(other, (int, float)):
```
Distance class should support both numerical types